### ISSUE TITLE
V1 expansion module API does't support more than one

### ISSUE THREAD

**Issue by squinkylabs** [VCVRack/Rack#1243]
(two new Squinky Labs modules are on hold because of this bug).

I want to make a module that can talk to expanders on both sides.

I find if I put this module in between two other modules that process does not get called with both `Module::leftModule` and `Module::rightModule` set to point to the sibling modules.

Instead, it appears to alternate. One process call `leftModule` will be set, on the next call `rightModule` will bet set. It does not seem random - they seem to strictly alternate.

Actually, that isn't always true. Picking some other modules to go on either side I do not get the alternating behavior. But so far I have been unable to get a module to be called with both sides active.

**Comment by MarcBoule** [comment]
I'm noticing something similar, it might be related. Although I am not planning to have expanders on both sides of a module, I did test it out, and it does seem like there may be an issue with simultaneously wanting to check (and use) `leftModule` and `rightModule`. Here is a test case that shows the potentiel problem.

I add the following code to my module's `process()` method (_GateSeq64_ in this case), expecting to see no output in the console when I put copies of the exapander on both sides.

```c++
	if (!(rightModule && rightModule->model == modelGateSeq64Expander)) 
		INFO("Right missing");
	if (!(leftModule && leftModule->model == modelGateSeq64Expander)) 
		INFO("Left missing");
```

Unfortunately it continually shows that the left module is missing.

![image](https://user-images.githubusercontent.com/33604154/55889189-aee8a580-5b7e-11e9-888f-35883444fdd4.png)

**Comment by squinkylabs** [comment]
Thanks for adding that, @MarcBoule . I should have provided some source like you did! Here's my current stripped down test. Ever so slightly more stripped down.
```inline void FooModule::process(const ProcessArgs &args)
{
    printf("rm=%d lm=%d\n", bool(rightModule), bool(leftModule));
    fflush(stdout);
}```

**Comment by AndrewBelt** [comment]
Fixed in 49e48ba
